easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-4.5.1-foss-2016a-METIS-5.1.0.eb

```diff
@@ -0,0 +1,24 @@
+name = 'SuiteSparse'
+version = '4.5.1'
+
+homepage = 'http://faculty.cse.tamu.edu/davis/suitesparse.html'
+description = """SuiteSparse is a collection of libraries manipulate sparse matrices."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'opt': True, 'unroll': True, 'pic': True}
+
+source_urls = ['http://faculty.cse.tamu.edu/davis/SuiteSparse/']
+sources = [SOURCE_TAR_GZ]
+
+metis = 'METIS'
+metis_ver = '5.1.0'
+versionsuffix = '-%s-%s' % (metis, metis_ver)
+dependencies = [(metis, metis_ver)]
+
+preconfigopts = 'LAPACK="$LIBLAPACK" ' 
+prebuildopts = 'LAPACK="$LIBLAPACK" ' 
+preinstallopts = 'LAPACK="$LIBLAPACK" ' 
```

Comment:
@boegel @hajgato @jrbosch
After a quick check, it appears the SuiteSparse libraries are linked to the right BLAS/LAPACK. I ran `ldd libumfpack.so` (which is built by SuiteSparse) and it is correctly linked to OpenBLAS or IntelMLK respectively if I use `foss` or `ictce` toolchain.

So, there is no need to change #863 and these 3 lines can be safely removed from the easyconfig file:
```
preconfigopts = 'LAPACK="$LIBLAPACK" ' 
prebuildopts = 'LAPACK="$LIBLAPACK" ' 
preinstallopts = 'LAPACK="$LIBLAPACK" ' 
```